frontend/src/components/pipelines-or-deployments/log-modal/LogsModal.jsx

```diff
@@ -40,7 +41,7 @@ const LogsModal = ({
       .then((res) => {
         const logDetails = res.data.results.map((item) => ({
           id: item.id,
```

Comment:
Can we add optional chaining wherever required.

Reply:
@jagadeeswaran-zipstack Thanks for the catch. Done. I have added optional chaining in places where it was necessary.